stdio/Cargo.toml

```diff
@@ -10,15 +10,14 @@ repository = "https://github.com/paritytech/jsonrpc"
 version = "16.0.0"
 
 [dependencies]
-futures = { version = "0.3", features = [ "compat" ] }
+futures = "0.3"
 jsonrpc-core = { version = "16.0", path = "../core" }
 log = "0.4"
-tokio = "0.1.7"
-tokio-codec = "0.1.0"
-tokio-io = "0.1.7"
-tokio-stdin-stdout = "0.1.4"
+tokio = { version = "0.2", features = ["io-std", "io-driver", "io-util"] }
+tokio-util = { version = "0.3", features = ["codec"] }
 
 [dev-dependencies]
+tokio = { version = "0.2", features = ["rt-core", "macros"] }
```

Comment:
these `dev-dependecues` will leaked into the `release` build, however https://github.com/rust-lang/cargo/pull/8997 have been stabalized so should be ready soon I guess.